docs/api.md

```diff
@@ -401,7 +401,7 @@ True if `type` is a pointer to a struct.
 
 ---
 
-    function types.type:ispointertofunction()
+    types.type:ispointertofunction()
```

Comment:
should this actually be `type:ispointertofunction()` (i.e. no `types.`)

Reply:
Yes, I concur and changed it